sympy/concrete/tests/test_sums_products.py

```diff
@@ -1053,6 +1053,10 @@ def test_is_convergent():
     # issue 19545
     assert Sum(1/n - 3/(3*n +2), (n, 1, oo)).is_convergent() is S.true
 
+    # issue 19836
+    assert Sum(4/(n + 2) - 5/(n + 1) + 1/n,(n,7,oo)).is_convergent() is S.true
```

Comment:
Is this test related to the other change (about `is_zero`)?

Reply:
Actually I am new to this and I made a mess that I know. So can I make another PR that just removing `expr*0 == S.Zero` from the code?

Reply:
It's probably best to do that. Otherwise you can reuse this PR with:
```
$ git reset HEAD^
< check git diff, undo the unrelated change to the code >
$ git add <filename>
$ git commit
$ git push --force
```

Reply:
I have done the changes in this PR.